### PR TITLE
Change the confusing function name from chat_event_cb to speech_event_cb

### PR DESCRIPTION
**Pull request by yantrabuddhi**
fixed by wenwei

### CODE REVIEW

File: src/ros_commo.py [hunk 1]
```diff
@@ -308,30 +308,19 @@ def chat_perceived_text_cb(self, chat_heard):
 	# started, or stopped vocalizing.  This message might be published
 	# by either the TTS module itself, or by some external chatbot.
 	#
-	# Unit test by saying
-	#    rostopic pub --once chat_events std_msgs/String speechstart
-	#    rostopic pub --once chat_events std_msgs/String speechend
```
Comment (linas) [inline src/ros_commo.py:313]: Can you restore these comments? They are useful for debugging, to manually verify that the system is working correctly.

File: src/ros_commo.py [hunk 2]
```diff
@@ -308,30 +308,19 @@ def chat_perceived_text_cb(self, chat_heard):
...
+		elif speech_event.data == "stop":
+			rospy.loginfo("ending speech")
 			self.puta.vocalization_ended()
-			rospy.loginfo("webui ending speech")
-
-		elif chat_event.data == "listen_start":
-			self.puta.listening_started()
-			rospy.loginfo("webui ending speech")
-
-		elif chat_event.data == "listen_stop":
```
Comment (linas) [inline src/ros_commo.py:328]: so .. what happened here? Are these being removed, because they are no longer published?

File: src/ros_commo.py [hunk 3]
```diff
@@ -308,30 +308,19 @@ def chat_perceived_text_cb(self, chat_heard):
...
-			self.puta.listening_ended()
-			rospy.loginfo("webui ending speech")
-
-
+		elif speech_event.data.startswith("duration"):
```
Comment (linas) [inline src/ros_commo.py:319]: ?? Is something actually encoding duration as a string?  What component does this? 

More abstractly, how is it even possible to know what the duration will be? Isn't this a prediction about the future?

File: src/ros_commo.py (hunk 2 above)
Comment (yantrabuddhi) [reply src/ros_commo.py:328]: I am a bit in doubt, the chat_events subscribes to speech_events and everywhere it mentions speech_events, speech_events only publishes start and stop and duration

File: src/ros_commo.py (hunk 3 above)
Comment (yantrabuddhi) [reply src/ros_commo.py:319]: its not supposed to be a prediction its like elapsed time start to stop i guess. @wenwei-dev could clarify

File: src/ros_commo.py (hunk 1 above)
Comment (yantrabuddhi) [reply src/ros_commo.py:313]: i restored them, but strangely its speech_events not chat_events?

File: src/ros_commo.py (hunk 2 above)
Comment (linas) [reply src/ros_commo.py:328]: OK, well, I guess the API must have been changed, som-when

**Pull request merged**